source/rest-v2.md

```diff
@@ -697,6 +976,21 @@ The JSON response returns a list of collections with the following attributes:
 | Turkish    | tr   |
 | Ukranian   | uk   |
 
+# Extraction Options
+
+The following fields defines options to extract hotspots from a PDF
+
+| Name                | Type   | Required | Description                                                                                                                                       |
+| ------------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| links               | Object | No       | Extracts link hotspots for any URLs, document links, and email addresses found in PDF file (see object definition table bellow)                   |
```

Comment:
`Extracts link hotspots from any URLs, document links, and email addresses found in the PDF file (see object definition table below)`